airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -0,0 +1,22 @@
+import { LoadingPage } from "components";
+import { Navigate } from "react-router-dom";
+import { CloudRoutes } from "../cloudRoutes";
+import {
+  useListCloudWorkspaces,
+  useWorkspaceService,
+} from "../services/workspaces/WorkspacesService";
+
+export const DefaultView: React.FC = () => {
+  const workspaces = useListCloudWorkspaces();
+  const { selectWorkspace } = useWorkspaceService();
+
+  // Only show the workspace creation list if there is more than one workspace
+  if (workspaces.length > 1) {
```

Comment:
Isn't it better to put 1 `useEffect` with condition instead of rendering in 1 case and calling selectWorkspace in another? It feels a bit unnatural to call sideEffects directly during rendering without any `useEffect` call.